runners/flink/pom.xml

```diff
@@ -93,7 +93,6 @@
                     org.apache.beam.sdk.testing.LargeKeys$Above100MB,
                     org.apache.beam.sdk.testing.UsesCommittedMetrics,
                     org.apache.beam.sdk.testing.UsesTestStream,
-                    org.apache.beam.sdk.testing.UsesSplittableParDo
```

Comment:
Add a comment why the batch execution still doesn't allow it; or remove this exclusion from the batch execution too?